docs/Antora/modules/knowledge_base/pages/signals.adoc

```diff
@@ -119,7 +122,7 @@ xref:packets.adoc[Packets] will not contain any data.
 To obtain data from Packets of Signals with an `Implicit` Data Rule, the values are
 calculated depending on the Rule type. Currently, two rules are available in openDAQ:
 
-* **Linear Rule**: The Rule parameters include a `delta` and `start` numbers. The values are calculated according to the following equation: `PacketOffset + SampleIndex * delta + start`, where the `PacketOffset` is available on the Packet carrying the data and the `SampleIndex` represents the index of the data entry being calculated within a given packet.
+* **Linear Rule**: The Rule parameters include a `delta` and `start` numbers. The values are calculated according to the following equation: `ReferenceDomainOffset + PacketOffset + SampleIndex * delta + start`, where the `PacketOffset` is available on the Packet carrying the data and the `SampleIndex` represents the index of the data entry being calculated within a given packet. `ReferenceDomainOffset` is only applied if available in Reference Domain Info of the Domain Signal's Data Descriptor.
```

Comment:
Similarly as above, offset does not only apply to the linear rule (it's not part of it).